content/docs/welcome-guide/requirements.md

```diff
@@ -225,6 +225,7 @@ O3DE also requires some additional library packages to be installed:
 * mesa-common-dev
 * libssl-dev
 * libunwind-dev
+* libzstd-dev
```

Comment:
You should also add this to the `sudo` shell command, a few lines after this list.

Reply:
good catch!